frontend/src/app/sessions/new-readonly-session-dialog/new-readonly-session-dialog.component.html

```diff
@@ -4,17 +4,17 @@
  -->
 
 <form (ngSubmit)="requestSession()">
```

Comment:
In the `new-readonly-session-dialog.component.ts` the old `legacy-dialog` is still used and therefore requesting a readonly session is not possible right now. In addition, you need to add the `dialog` class here to add some padding to the dialog.